cfonts/core.py

```diff
@@ -354,7 +355,12 @@ def render(
         if output:
             output[0] = style.open + output[0]
             output[-1] += style.close
-    return "\n".join(output)
+    if raw:
+        # replace the escape character with the bytes for the escape character
+        # \033 also works
+        return "\n".join(output).replace("\x1b", "\\x1b")
```

Comment:
```suggestion
        return repr("\n".join(output))
```

This would be better in case there are other escape sequences in the string

Reply:
Though I personally find it messier, since everything is one one line, this does work just fine. I'm not sure what other characters could be in the string, that would need to be escaped. Maybe do this, but then replace `\r` and `\n` with actual newlines?